.github/workflows/semantic.yml

```diff
@@ -0,0 +1,21 @@
+name: "Lint PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - uses: ./.github/actions/action-semantic-pull-request
+        env:
```

Comment:
APISIX doesn't use the default configuration. Let's move the configuration in https://github.com/apache/apisix/blob/master/.github/semantic.yml to the new action.

Reply:
> APISIX doesn't use the default configuration. Let's move the configuration in https://github.com/apache/apisix/blob/master/.github/semantic.yml to the new action.

Is the `allowRevertCommits: true` means accepting a format like `Revert "feat(xxx): xxx"`

Reply:
Yes

Reply:
I add the custom types, and action-semantic-pull-request does not support the option `titleOnly` and `allowRevertCommits`, I tested and found that action-semantic-pull-request check title only defaultly.